mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRRestAPIObjectsHelper.java

```diff
@@ -146,11 +152,21 @@ public static Response showMCRObject(String pathParamId, String queryParamStyle,
         StringWriter sw = new StringWriter();
         XMLOutputter outputter = new XMLOutputter(Format.getPrettyFormat());
         try {
-            outputter.output(doc, sw);
+            if (MCRConfiguration.instance().getString("MCR.ContentTransformer.RestApiXmlFilter.Stylesheet", "").length() > 0) {
```

Comment:
Do not hardcode any transformer configuration here. It's better to make a filter property that defines a ContentTransformer like in my original review.
Simply add something like `MCR.RestAPI.v1.Filter.XML=RestApiXmlFilter` and use only `MCR.RestAPI.v1.Filter.XML` in your code to detect IF there is a content transformer defined.
ContentTransfomer MAY use Stylesheets but can be of any kind...